comment/tests/test_utils.py

```diff
@@ -98,6 +110,15 @@ def test_unauthenticated_user(self):
         self.request.user = AnonymousUser()
         self.assertIsNone(get_user_for_request(self.request))
 
+    def test_user_for_request(self):
+        request = self.factory.get('/')
+        request.user = AnonymousUser()
+        # test unauthenticated user
+        self.assertIsNone(get_user_for_request(request))
```

Comment:
anonymous user is already tested in the function above: https://github.com/Radi85/Comment/pull/201/files#diff-c218deacbe7b0c004243620d56d658670d0cb36bb5aa08a6d1bc19f6909761b7L97